<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profileSetupDlg</class>
 <widget class="QDialog" name="profileSetupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup profiles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Profiles</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="newProfileBtn">
           <property name="text">
            <string>Create New Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>or</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectProfileBtn">
           <property name="text">
            <string>Select Existing Profile</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="profileFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="7" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <widget class="QLineEdit" name="timestampFormatEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="timestampFormatBtn">
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>?</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLineEdit" name="rightCamPathEdit"/>
              </item>
             </layout>
            </item>
            <item row="10" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QComboBox" name="rotateLeftBox"/>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>degrees</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>RotateRightImage</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>ImageTimestampStart</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Profile description</string>
              </property>
             </widget>
            </item>
            <item row="11" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QComboBox" name="rotateRightBox"/>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>degrees</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="8" column="2">
             <widget class="QLineEdit" name="timestampStartEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>RightCameraImagePath</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>ImageTimestampType</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>CollectMetadata</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <widget class="QLineEdit" name="calFilePathEdit"/>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>CalibrationFileName</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>ImageTimestampFormat</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Profile name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="profileDescriptionEdit"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>FrameNumberIndices</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLineEdit" name="leftCamPathEdit"/>
              </item>
             </layout>
            </item>
            <item row="6" column="2">
             <widget class="QComboBox" name="timestampTypeBox"/>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="profileNameEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>LeftCameraImagePath</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>RotateLeftImage</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="stFrameIndEdit"/>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>to</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="endFrameIndEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="frameIndHelpBtn">
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>?</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="9" column="2">
             <widget class="QGroupBox" name="metadataInfoBox">
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QRadioButton" name="metadataYesBtn">
                 <property name="text">
                  <string>Yes</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="metadataNoBtn">
                 <property name="text">
                  <string>No</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>SubsampleMask</string>
              </property>
             </widget>
            </item>
            <item row="12" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLineEdit" name="subsampleMaskEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="subsampleMaskBtn">
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>?</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Species Collections</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="newSpcCollectionBtn">
           <property name="text">
            <string>Create new 
 Species Collection</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>or</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectSpcCollectionBtn">
           <property name="text">
            <string>Select Existing
 Species Collection</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="speciesFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Species Collection Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Species Collection Description</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="spcCollectionNameEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="spcCollectionDescEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="speciesView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>351</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="speciesInsertBtn">
              <property name="text">
               <string>Insert</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="speciesEditBtn">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="speciesDeleteBtn">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Metadata Groups</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="newMetadataGroupBtn">
           <property name="text">
            <string>Create new 
 Metadata Group</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>or</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectMetadataGroupBtn">
           <property name="text">
            <string>Select Existing 
Metadata Group</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="metadataFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Metadata Group</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string> Description</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="metadataGroupEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="metadataGroupDescEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="metadataView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>351</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="metadataInsertBtn">
              <property name="text">
               <string>Insert</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="metadataEditBtn">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="metadataDeleteBtn">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="reloadBtn">
       <property name="text">
        <string>Reload deployment with new profile</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>newProfileBtn</tabstop>
  <tabstop>selectProfileBtn</tabstop>
  <tabstop>profileNameEdit</tabstop>
  <tabstop>profileDescriptionEdit</tabstop>
  <tabstop>leftCamPathEdit</tabstop>
  <tabstop>rightCamPathEdit</tabstop>
  <tabstop>calFilePathEdit</tabstop>
  <tabstop>stFrameIndEdit</tabstop>
  <tabstop>endFrameIndEdit</tabstop>
  <tabstop>timestampTypeBox</tabstop>
  <tabstop>timestampFormatEdit</tabstop>
  <tabstop>timestampStartEdit</tabstop>
  <tabstop>metadataYesBtn</tabstop>
  <tabstop>metadataNoBtn</tabstop>
  <tabstop>rotateLeftBox</tabstop>
  <tabstop>rotateRightBox</tabstop>
  <tabstop>timestampFormatBtn</tabstop>
  <tabstop>newSpcCollectionBtn</tabstop>
  <tabstop>selectSpcCollectionBtn</tabstop>
  <tabstop>frameIndHelpBtn</tabstop>
  <tabstop>spcCollectionNameEdit</tabstop>
  <tabstop>spcCollectionDescEdit</tabstop>
  <tabstop>speciesView</tabstop>
  <tabstop>speciesInsertBtn</tabstop>
  <tabstop>speciesEditBtn</tabstop>
  <tabstop>speciesDeleteBtn</tabstop>
  <tabstop>newMetadataGroupBtn</tabstop>
  <tabstop>selectMetadataGroupBtn</tabstop>
  <tabstop>metadataGroupEdit</tabstop>
  <tabstop>metadataGroupDescEdit</tabstop>
  <tabstop>metadataView</tabstop>
  <tabstop>metadataInsertBtn</tabstop>
  <tabstop>metadataEditBtn</tabstop>
  <tabstop>metadataDeleteBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
  <tabstop>saveBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
